cmd/abci-cli/abci-cli.go

```diff
@@ -287,6 +300,53 @@ func persistentArgs(line []byte) []string {
 
 //--------------------------------------------------------------------------------
 
+func or(err1 error, err2 error) error {
+	if err1 == nil {
+		return err2
+	} else {
+		return err1
+	}
+}
+
+func cmdTest(cmd *cobra.Command, args []string) error {
+	fmt.Println("Running tests")
+
+	var err error
+
+	err = servertest.InitChain(client)
+	fmt.Println("")
+	err = or(err, servertest.SetOption(client, "serial", "on"))
+	fmt.Println("")
+	err = or(err, servertest.Commit(client, nil))
+	fmt.Println("")
+	err = or(err, servertest.DeliverTx(client, []byte("abc"), code.CodeTypeBadNonce, nil))
+	fmt.Println("")
+	err = or(err, servertest.Commit(client, nil))
+	fmt.Println("")
+	err = or(err, servertest.DeliverTx(client, []byte{0x00}, code.CodeTypeOK, nil))
+	fmt.Println("")
+	err = or(err, servertest.Commit(client, []byte{0, 0, 0, 0, 0, 0, 0, 1}))
+	fmt.Println("")
+	err = or(err, servertest.DeliverTx(client, []byte{0x00}, code.CodeTypeBadNonce, nil))
+	fmt.Println("")
+	err = or(err, servertest.DeliverTx(client, []byte{0x01}, code.CodeTypeOK, nil))
+	fmt.Println("")
+	err = or(err, servertest.DeliverTx(client, []byte{0x00, 0x02}, code.CodeTypeOK, nil))
+	fmt.Println("")
+	err = or(err, servertest.DeliverTx(client, []byte{0x00, 0x03}, code.CodeTypeOK, nil))
+	fmt.Println("")
+	err = or(err, servertest.DeliverTx(client, []byte{0x00, 0x00, 0x04}, code.CodeTypeOK, nil))
+	fmt.Println("")
+	err = or(err, servertest.DeliverTx(client, []byte{0x00, 0x00, 0x06}, code.CodeTypeBadNonce, nil))
+	fmt.Println("")
```

Comment:
can these `fmt.Println("")` be handled in `func or()` ?

Reply:
I'd prefer not to.